### PR TITLE
Client init

### PR DESCRIPTION
**Pull request by Whiteaj36**
There was some server changes that I made that I didn't know got made, just check them over. I ran rake but I'm not certain if that is what caused this.

Specifically in the gemfile.

    +    sqlite3 (1.3.10)   //added this
         sqlite3 (1.3.10-x86-mingw32)
         thor (0.19.1)
        thread_safe (0.3.4)
     @@ -106,6 +107,7 @@ GEM
           json (>= 1.8.0)
     
     PLATFORMS
    +  ruby //added this
       x86-mingw32
 
This comment is also me practicing markdown. I know you can just look at the file and see what I added.

### CODE REVIEW

**Comment by robwagner33** [comment]
`This comment is also me practicing markdown. I know you can just look at the file and see what I added.`

I like you.

File: Client/DrawThis/DrawThis/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (robwagner33) [inline Client/DrawThis/DrawThis/Base.lproj/Main.storyboard]: This is generated right? you aren't maintaining this

File: Client/DrawThis/DrawThis/TitleScreenViewController.h [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+//  Copyright (c) 2015 Alexander J White. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface TitleScreenViewController : UIView
+
+@property (weak, nonatomic) IBOutlet UIButton *startButton;
+
+@property (nonatomic, assign) BOOL didReceiveMessage;
```
Comment (robwagner33) [inline Client/DrawThis/DrawThis/TitleScreenViewController.h]: Is this the naming style for booleans? isSomething hadSomethingHappen didRecieveMessage

File: Client/DrawThis/DrawThis/TitleScreenViewController.m [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+
+- (IBAction)didPressStartButton:(id)sender {
+    if (self.didReceiveMessage) {
+        [self.startButton setTitle:@"Click to Receive Message" forState:UIControlStateNormal];
+        self.didReceiveMessage = NO;
+    }
+    else {
+        [self.startButton setTitle:@"Message received" forState:UIControlStateNormal];
+        self.didReceiveMessage = YES;
```
Comment (robwagner33) [inline Client/DrawThis/DrawThis/TitleScreenViewController.m]: YES and NO, haha love it

File: Server/Gemfile.lock [hunk 4]
```diff
@@ -91,6 +91,7 @@ GEM
       actionpack (>= 3.0)
```
Comment (robwagner33) [inline Server/Gemfile.lock]: We shouldnt version this. Lockfile is garbage if we are using 2 different OSes

**Comment by robwagner33** [comment]
Looks good. Merging. I'll take out lock in the next push.

File: Client/DrawThis/DrawThis/Base.lproj/Main.storyboard (hunk 1 above)
Comment (Whiteaj36) [inline Client/DrawThis/DrawThis/Base.lproj/Main.storyboard]: Yes, storyboard files are completely generated. We should talk about a maintenance plan for these as well going forward

Comment (robwagner33) [inline Client/DrawThis/DrawThis/Base.lproj/Main.storyboard]: :+1:

File: Client/DrawThis/DrawThis/TitleScreenViewController.h (hunk 2 above)
Comment (Whiteaj36) [inline Client/DrawThis/DrawThis/TitleScreenViewController.h]: Yes it is, objective c is meant to be read like a sentence. "On Server Response" functions are often structured liked so

    -(void)getHelloDidCompleteWithResponse:(ResponseType) response andOtherVariableForExample:(NSString) foo

File: Client/DrawThis/DrawThis/TitleScreenViewController.m (hunk 3 above)
Comment (Whiteaj36) [inline Client/DrawThis/DrawThis/TitleScreenViewController.m]: Boolean values!

File: Server/Gemfile.lock (hunk 4 above)
Comment (Whiteaj36) [inline Server/Gemfile.lock]: Want to update the gitignore with this?

**Pull request merged**